src/components/Contacts/ContactDetails/ContactDontationsTab/PartnershipInfo/PartnershipInfo.tsx

```diff
@@ -0,0 +1,187 @@
+            ? DateTime.fromISO(contact?.pledgeStartDate).toLocaleString()
+            : t('No Date Available')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <DateRangeOutlined color="disabled" style={{ margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Last Gift')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.lastDonation?.donationDate
+            ? DateTime.fromISO(
+                contact?.lastDonation.donationDate,
+              ).toLocaleString()
+            : t('No Date Available')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Lifetime Gifts')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {currencyFormat(
+            contact?.totalDonations ?? 0,
+            contact?.pledgeCurrency ?? 'USD',
+          )}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Referred by ')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.contactReferralsToMe.nodes
+            .map((referral) => {
+              return referral.referredBy.name;
+            })
+            .toLocaleString()}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Send Appeals')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.noAppeals ? t('No') : t('Yes')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Next Increase Ask')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1" role="textbox">
+          {DateTime.fromISO(contact?.nextAsk ?? '').toLocaleString()}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <Divider />
+      <PartnershipTitle variant="h6">{t('Partner Account')}</PartnershipTitle>
+      {contact?.contactDonorAccounts.nodes.map((donor) => {
+        return (
+          <IconAndTextContainerCenter key={donor.id}>
```

Comment:
Will this partner info piece be shown anywhere else? Just wondering if a component would help encase this well if that was the case.

Reply:
As far as I know this will not be repeated.